server_py/flatgov/flatgov/dev.py

```diff
@@ -9,15 +9,15 @@
 
 
 DATABASES = {
```

Comment:
We need to figure out what to do with dev.py. If you prefer to have sqlite as the dev backend, we should remove dev.py from git; I'm using postgresql.